packages/browsers/test/src/versions.ts

```diff
@@ -6,6 +6,6 @@
 
 export const testChromeBuildId = '127.0.6533.72';
 export const testChromiumBuildId = '1083080';
-export const testFirefoxBuildId = '130.0a1';
+export const testFirefoxBuildId = 'stable_129.0';
```

Comment:
probably local copy for the test server needs to be changed as well

Reply:
although I expected everything to just work

Reply:
looks like a bug in resolveDownloadPath in firefox.ts We consider any version to be downloaded into archiveNightly() location

Reply:
That should be only for nightly, no? When we give `stable_` prefix we should never hit it.

Reply:
resolveDownloadPath does not consider the prefix yet but it should be updated to return different results based on the prefix, i.e., use archive() instead of archiveNightly() for non-nightly prefixes.